Source/misdat.cpp

```diff
@@ -248,4 +248,22 @@ void MissileFileData::LoadGFX()
 	}
 }
 
+void InitMissileGFX(bool loadHellfireGraphics)
+{
+	for (size_t mi = 0; MissileSpriteData[mi].animFAmt != 0; mi++) {

```

Comment:
While technically correct, don't you think using `size_t` here is a bit overkill? This is the type of code that I think damages readability.

Would just use `int` here as the previous code did.

Reply:
Happy to change it back given the comparison with a `missile_graphic_id` member. The main reason I wanted to use `size_t` here was so I can see from that line that `mi` is used as an index and not stored/saved like what happens with `ActiveMissiles` and the other common use of `mi`.

Reply:
I'm fine either way.